helpers/securewebhook/securewebhook.go

```diff
@@ -1,14 +1,33 @@
 package securewebhook
```

Comment:
part of me thinks we should call the package something more generic like eventsettings since event webhook is under event settings in mail settings

or we could call the entire package eventwebhook

Reply:
the file name makes sense but from an organization perspective, it will help in case we have other files related to event webhook or event settings

Reply:
good idea, I am for it

Reply:
👍